@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageABRHandleControl.sqf

```diff
@@ -0,0 +1,36 @@
+params ["_heli", "_mpdIndex", "_control", "_state", "_persistState"];
+
+#include "\fza_ah64_mpd\headers\tsd.hpp"
+#include "\fza_ah64_mpd\headers\mfdConstants.h"
+
+private _abrPage        = _state get "pageNumber";
+private _tsdState      = _state get "tsdState";
+
+switch (_control) do {
+    case "t2": {
+        [_heli, _mpdIndex, "ase"] call fza_mpd_fnc_setCurrentPage;
+    };
+    case "t4": {
+        [_heli, _mpdIndex, "tsd", _tsdState] call fza_mpd_fnc_setCurrentPage;
+    };
+    case "b1": {
+        [_heli, _mpdIndex, "menu"] call fza_mpd_fnc_setCurrentPage;
+    };
+    case "b2": {
+        if (_abrPage > 1) then {_abrPage = _abrPage - 1;};
+        _state set ["pageNumber", _abrPage];
+    };    
+    case "b3": {
+        if (_abrPage < 3) then {_abrPage = _abrPage + 1;};
+        _state set ["pageNumber", _abrPage];
+    };
+    case "b4": {
+        [_heli, _mpdIndex, "tsd", ["subPageVarPage"] createHashMapFromArray [TSD_WPT]] call fza_mpd_fnc_setCurrentPage;
+    };
+    case "b5": {
+        [_heli, _mpdIndex, "tsd", ["subPageVarPage"] createHashMapFromArray [TSD_RTE]] call fza_mpd_fnc_setCurrentPage;
+    };
+    case "b6": {
+        [_heli, _mpdIndex, "tsd", ["subPageVarPage"] createHashMapFromArray [TSD_THRT]] call fza_mpd_fnc_setCurrentPage;
```

Comment:
You use both the binary and unary variants of `createHashMapFromArray`, please can you standardise on one in this PR?

Generally i have preferred the unary variant, as then you don't have two different arrays that could theoretically have different lengths